### ISSUE TITLE
`make` fails if no `WRITEOPENSSHKEYS=1` given

### ISSUE THREAD

**Issue by lez** [pts/pts-dropbear#2]
We are very grateful that you added ed25519 support to dropbear.
However, the default make options seem to be not enough anymore:
```
| ../dropbear-2017.75/dropbearkey.c: In function 'dropbearkey_main':
| ../dropbear-2017.75/dropbearkey.c:305:51: error: 'format' undeclared (first use in this function)
|   if (signkey_generate(keytype, bits, filename, 0, format) == DROPBEAR_FAILURE)
|                                                    ^~~~~~
| ../dropbear-2017.75/dropbearkey.c:305:51: note: each undeclared identifier is reported only once for each function it appears in
```
Adding `WRITEOPENSSHKEYS=1` solves this compile error.

**Comment by pts** [comment]
Thank you for reporting this! Fixed it in 432c3f9dd5d51988c6d09a4ee73024de59707234.

**Comment by lez** [comment]
Wow! Thanks for the instant fix! Really appreciate it!
